import java.text.NumberFormat;
import java.text.ParseException;


public class ExemploExcecao {
    public static void main(String[] args) throws ParseException {
        //Number valor = Double.valueOf("a1.75");
        Number valor;
        try {
            valor = NumberFormat.getInstance().parse("a1.75");
            System.out.println(valor);

        } catch (ParseException e) {
            System.out.println("Caracter inválido na string");
          }
    }
}
